### PR TITLE
feat: frontend cleanup

### PR DESCRIPTION
**Pull request by tobySolutions**
- Remove the “My account” from the sign in page.
- ⁠Make sure the logo navigates to the home page on the dashboard.
- ⁠update max words for title from 10 to 30 
- ⁠add loader to signin page when callback returns

### CODE REVIEW

File: packages/frontend/src/routes/signIn/index.tsx [hunk 1]
```diff
@@ -38,17 +37,18 @@ function SignIn() {
...
   storeDataInCookie("userCode", userCode, 1);
 
   useEffect(() => {
     const userCodeFromCookie = getDataInCookie("userCode");
-
     const isGoogleInUrl = window.location.href.includes("google")
       ? "google"
       : "github";
 
+    console.log(userCodeFromCookie);
```
Comment (chibuike-19) [inline packages/frontend/src/routes/signIn/index.tsx:48]: Is there a particular reason for storing the loading in cookies? Maybe you can just add a loading state that you can toggle when making the request? 
Also can you remove the console.log

File: packages/frontend/src/routes/signIn/index.tsx [hunk 2]
```diff
@@ -136,27 +146,29 @@ function SignIn() {
               <button
                 type="button"
                 onClick={handleGoogleSignIn}
-                className="flex items-center justify-center w-full px-4 py-2 text-white bg-blue-600 rounded-md hover:bg-blue-700 focus:outline-none focus:ring-2 focus:ring-offset-2 focus:ring-red-500"
+                disabled={loading}
+                className="flex items-center justify-center w-full px-4 py-2 text-white bg-blue-600 rounded-md hover:bg-blue-700 focus:outline-none focus:ring-2 focus:ring-offset-2 focus:ring-red-500 disabled:opacity-50"
               >
-                <FaGoogle className="mr-2" />
+                {oauthMethod === "google" && loading ? (
+                  <LoadingIcon className="w-8 h-8 animate-spin text-primary-white" />
```
Comment (chibuike-19) [inline packages/frontend/src/routes/signIn/index.tsx:153]: Can we maintain the same loader used across the app?

Comment (tobySolutions) [reply packages/frontend/src/routes/signIn/index.tsx:153]: Where is the Loader's name?

File: packages/frontend/src/routes/signIn/index.tsx (hunk 1 above)
Comment (tobySolutions) [reply packages/frontend/src/routes/signIn/index.tsx:48]: On it!

**Review by chibuike-19 — APPROVED** [review]
LGTM!

**Pull request merged**